Sources/Plasma/FeatureLib/pfPython/plPythonSDLModifier.cpp

```diff
@@ -314,6 +316,32 @@ void plPythonSDLModifier::IDirtySynchState(const ST::string& name, bool sendImme
     }
 }
 
+template<typename T>
+static std::optional<T> IConvertPythonNumber(PyObject* o)
+{
+    if constexpr (std::is_same_v<T, int>) {
+        if (PyLong_Check(o))
+            return PyLong_AsLong(o);
+        if (PyNumber_Check(o)) {
+            pyObjectRef pyLong = PyNumber_Long(o);
+            return PyLong_AsLong(pyLong.Get());
+        }
+        return std::nullopt;
+    } else if constexpr (std::disjunction_v<std::is_same<T, float>, std::is_same<T, double>>) {
```

Comment:
I'm fairly certain you can just
```suggestion
    } else if constexpr (std::is_same_v<T, float> || std::is_same_v<T, double>) {
```

Reply:
Yeah, that worked with MSVC... I just wasn't sure why gcc and clang were exploding, but you've clarified that nicely :)